webpack/animateHeader.js

```diff
@@ -0,0 +1,147 @@
+import anime from 'animejs'
+
+// Detect request animation frame
+var scroll =
+  window.requestAnimationFrame ||
+  window.webkitRequestAnimationFrame ||
+  window.mozRequestAnimationFrame ||
+  window.msRequestAnimationFrame ||
+  window.oRequestAnimationFrame ||
+  // IE Fallback, you can even fallback to onscroll
+  function(callback) {
+    window.setTimeout(callback, 1000 / 60);
+  };
+var lastPosition = -1;
+
+// my Variables
```

Comment:
This comment is notably from copy paste. "my variables". LOL

That said, it does not add anything. Please remove it.

Reply:
Sorry, I removed all of them